src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -618,6 +627,7 @@ class MessageHelperTest : public ::testing::Test {
   StringArray function_id_strings;
   StringArray events_id_strings;
   StringArray hmi_level_strings;
+  sync_primitives::Lock lock_;
```

Comment:
@LevchenkoS I suggest you to change the field names according to [Google C++ Style](https://google.github.io/styleguide/cppguide.html)